collision_objects:
  - id: 1
    type: 1
    name: "desk"
    dimension: [0.3, 0.3, 0.6]
    pose: [0, 0, -0.301, 0, 0, 0, 1]
    mesh_pkg_name: ""
    mesh_folder: ""
    mesh_file: ""
    visible: true
    dynamic: false
  - id: 6
    type: 1
    name: "selling_shelf1"
    dimension: [1.10, 0.50, 0.06]
    pose: [0, -0.85, 0.795, 0, 0, 0, 1]
    mesh_pkg_name: ""
    mesh_folder: ""
    mesh_file: ""
    visible: true
    dynamic: false
  - id: 9
    type: 1
    name: "selling_shelf_col1"
    dimension: [0.05, 0.50, 1.8]
    pose: [0.565, -0.85, 0.25, 0, 0, 0, 1]
    mesh_pkg_name: ""
    mesh_folder: ""
    mesh_file: ""
    visible: true
    dynamic: false
  - id: 10
    type: 1
    name: "selling_shelf_col2"
    dimension: [0.05, 0.50, 1.8]
    pose: [-0.565, -0.85, 0.25, 0, 0, 0, 1]
    mesh_pkg_name: ""
    mesh_folder: ""
    mesh_file: ""
    visible: true
    dynamic: false
  - id: 11
    type: 1
    name: "desk2"
    dimension: [1.20, 0.70, 0.05]
    pose: [0, 1.05, 0.245, 0, 0, 0, 1]
    mesh_pkg_name: ""
    mesh_folder: ""
    mesh_file: ""
    visible: true
    dynamic: false
